tools/Google.Cloud.Tools.ProjectGenerator/ApiMetadata.cs

```diff
@@ -130,7 +133,20 @@ private static void WriteProjectFile(string directory, XElement propertyGroup, X
                 doc = XElement.Load(file);
                 doc.Elements("Import").Where(x => (string)x.Attribute("Project") == @"..\..\StripDesktopOnNonWindows.xml").Remove();
                 doc.Elements("PropertyGroup").First().ReplaceWith(propertyGroup);
-                doc.Elements("ItemGroup").First().ReplaceWith(itemGroup);
+                doc.Elements("ItemGroup").First().ReplaceWith(dependenciesItemGroup);
```

Comment:
I think we can simplify this by putting the Exists condition on the Analyzer element instead of the ItemGroup element. Then we can still have a single ItemGroup.